src/chrome/content/rules/Conrad_Electronic.xml

```diff
@@ -0,0 +1,119 @@
+<!--
+	Invalid certificate:
+		produktinfo.conrad.com (expired)
+		www.produktinfo.conrad.com (expired, mismatch)
+		m.conrad.at (mismatch)
+  		m.conrad.nl (mismatch)
+  		m.conrad-electronic.nl (mismatch)
+  		m.conrad.pl (mismatch)
+		conrad.sk (mismatch)
+		www.conrad.sk (mismatch)
+		velkoobchod.conrad.sk (mismatch)
+		
+	Redirect to HTTP:
+		www.conrad.be
+		conrad.com
+		www.conrad.com
+		www.conrad.fr
+		www.conrad.it
+		business.conrad.it
+		www.conrad-electronic.co.uk
+
+	Refused:
+		conrad.at (redirect rule as workaround)
+		conrad.be
+		conrad.biz (redirect rule as workaround)
+		biz-conrad.ch (redirect rule as workaround)
+		m.conrad.cz
+		conrad.ch (redirect rule as workaround)
+  		conrad.de(redirect rule as workaround)
+		conrad.fr
+		conrad.hu (redirect rule as workaround)
+		conrad.it
+		conrad.nl (redirect rule as workaround)
+		conrad-electronic.co.uk
+
+	Timeout:
+		m.conrad.biz
+		m.conrad.ch
+		m.biz-conrad.ch
+		m.conrad.de
+-->
+
+<ruleset name="Conrad Electronic (partial)">
+	<target host="conrad.at" />
+	<target host="www.conrad.at" />
+	<target host="business.conrad.at" />
+	<target host="conrad.biz" />
+	<target host="www.conrad.biz" />
+	<target host="www.biz-conrad.ch" />
+	<target host="conrad.ch" />
+	<target host="www.conrad.ch" />
+	<target host="conrad.cz" />
+	<target host="www.conrad.cz" />
+	<target host="velkoobchod.conrad.cz" />
+	<target host="conrad.de" />
+	<target host="www.conrad.de" />
+	<target host="conradelektronik.dk" />
+	<target host="www.conradelektronik.dk" />
+	<target host="m.conradelektronik.dk" />
+	<target host="conrad.hr" />
+	<target host="www.conrad.hr" />
+	<target host="m.conrad.hr" />
+	<target host="conrad.hu" />
+	<target host="www.conrad.hu" />
+	<target host="conrad.nl" />
+	<target host="www.conrad.nl" />
+	<target host="conrad-electronic.nl" />
+	<target host="www.conrad-electronic.nl" />
+	<target host="conrad.pl" />
+	<target host="www.conrad.pl" />
+	<target host="conrad.se" />
+	<target host="www.conrad.se" />
+	<target host="m.conrad.se" />
+	<target host="conrad.si" />
+	<target host="www.conrad.si" />
+	<target host="m.conrad.si" />
+		
+	<target host="api.conrad.com" />
```

Comment:
This target returns a 404 on https. You need to find a valid test url. For the tests you need to add an exclusion for `^http://api\.conrad\.com/$`.

Reply:
Please see my comment above, I expect it to fail also. Or is my assumption wrong?

Reply:
Sure, you can comment it as `different content`, `status code mismatch` or similar.